modules/rpmrepo/manifests/install.pp

```diff
@@ -10,33 +10,12 @@
 # @example
 #   include rpmrepo::install
 class rpmrepo::install {
-  include httpd
 
-  file { '/var/www/html/repo':
-    ensure => directory,
-    mode   => '0644',
-  }
-
-  firewall::openport {'rpmrepo':
-    dports => '80',
-  }
-
-  $fpm_needs= ['ruby-devel', 'gcc', 'make', 'rpm-build', 'rubygems',]
-
-  package { $fpm_needs:
-    ensure => latest,
-    before => Exec[gem_update],
-  }
-
-  exec { 'gem_update':
-    command => 'gem update --system',
-    path    => '/usr/bin:/usr/sbin:/bin:/usr/local/bin',
-    before  => Package[fpm],
-  }
+  $tools = ['createrepo','yum-utils']
 
-  package { 'fpm':
-    ensure   => installed,
-    provider => 'gem',
-    #install_options => ['--no-ri', '--no-rdoc'],
+  $tools.each |String $tool | {
```

Comment:
there is no need to do loop over packages you can fo
package {["package1","package2"]: ensure =>....

https://www.puppetcookbook.com/posts/install-multiple-packages.html

Reply:
TNX. Already done.